### PR TITLE
feat: send emails to user and product approval

### PR DESCRIPTION
**Pull request by davidcormor**

### CODE REVIEW

File: src/infrastructure/adapters/database/repositories/utils.py [hunk 1]
```diff
@@ -5,30 +5,94 @@
...
+    data_response = request_to_ms_auth(jwt, uuid_user, True)
+    if data_response is not None:
+        email = data_response['email']
+    return email
+
+
+def send_email(subject: str, data: str, destination: list, is_html: bool = False) -> bool:
+    try:
+        type_content = "Html" if is_html else "Text"
+        url = "https://30mtgv2761.execute-api.us-east-2.amazonaws.com/dev/create"
```
Comment (diegocortes-cafeto) [inline src/infrastructure/adapters/database/repositories/utils.py:58]: is it possible to move this URL to properties file ?

File: src/infrastructure/adapters/database/repositories/utils.py [hunk 2]
```diff
@@ -5,30 +5,94 @@
...
 def default_prefix_cloud():
     path_datetime = str(datetime.today().strftime('%Y/month-%m/day-%d/%I-%M-%S'))
 
 
-def get_user_names(jwt, uuid_user) -> tuple:
-    first_name = 'undefined'
-    last_name = 'undefined'
-    url = f"https://meerkat-auth.herokuapp.com/api/v1/user/person/uuid/{uuid_user}"
+def request_to_ms_auth(jwt, uuid_user, get_person=False):
+    base_url = "https://meerkat-auth.herokuapp.com/api/v1/user"
```
Comment (diegocortes-cafeto) [inline src/infrastructure/adapters/database/repositories/utils.py:18]: is it possible to move the auth URL endpoint to properties file?

**Review by diegocortes-cafeto — APPROVED** [review]

File: src/infrastructure/adapters/database/repositories/utils.py (hunk 1 above)
Comment (davidcormor) [reply src/infrastructure/adapters/database/repositories/utils.py:58]: We have a configuration file called .flaskenv where we have all our environment variables. but EB has a limit of 50 variables. However, we going to change this part with AWS Systems Manager Parameter Store (BNS-147).

File: src/infrastructure/adapters/database/repositories/utils.py (hunk 2 above)
Comment (davidcormor) [reply src/infrastructure/adapters/database/repositories/utils.py:18]: We have a configuration file called .flaskenv where we have all our environment variables. but EB has a limit of 50 variables. However, we going to change this part with AWS Systems Manager Parameter Store (BNS-147).

Comment (diegocortes-cafeto) [reply src/infrastructure/adapters/database/repositories/utils.py:18]: sounds good, thanks

File: src/infrastructure/adapters/database/repositories/utils.py (hunk 1 above)
Comment (diegocortes-cafeto) [reply src/infrastructure/adapters/database/repositories/utils.py:58]: sounds good, thanks

**Pull request merged**